helm/loki/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: loki
 description: Helm chart for Grafana Loki in microservices mode
 type: application
-appVersion: 2.2.0
-version: 0.27.0
+appVersion: 2.2.1
+version: 0.32.0
```

Comment:
Not sure about the versioning here. Shouldn't this be the same as in our change log?

Reply:
yes. this will be changed by abs in ci though for the actual artifact installed on clusters